src/main/java/webserver/HandlerMapping.java

```diff
@@ -0,0 +1,34 @@
+package webserver;
+
+import controller.FileController;
+import domain.user.controller.UserController;
+
+import java.util.Arrays;
+import java.util.function.Function;
+
+public enum HandlerMapping {
+    GET_PAGE(new RequestLine(HttpMethod.GET, ".html"), FileController::getPage),
```

Comment:
```suggestion
    PAGE(HttpMethod.GET, ".html", FileController::getPage),
    CSS(HttpMethod.GET, ".css", FileController::getCss),
    USER_CREATE_PAGE(HttpMethod.GET, "/user/create", UserController::getCreateUser),
    USER_CREATE(HttpMethod.POST, "/user/create", UserController::postCreateUser);
```
이런 구조가 좀더 읽기에 자연스럽지 않을까요?
RequestLine 객체로 감쌀 필요는 없어보여요
from 메서드에서 적절한 enum 값을 찾을 때도 http method로 한 번 필터하고 path로 한 번 더 필터하면 되지 않을까요?

Reply:
말씀하신대로 수정했습니다!
훨씬 코드가 깔끔하고 가독성이 좋아진 것 같아요.

객체로 감쌀 때 한 번 더 고민해보는 습관가지도록 하겠습니다!